### DISCUSSION TITLE
Unable to submit nominations for stars.github.com

### DISCUSSION THREAD

**Discussion by logankilpatrick** [community/community#10274] [Other Feature Feedback, Questions, & Ideas]
Hey folks, for the last few months now, I have been unable to submit nominations to the GitHub stars website. 
<img width="1696" alt="Screen Shot 2022-01-18 at 6 08 47 PM" src="https://user-images.githubusercontent.com/35577566/150050251-50cb46d7-9fc6-49ea-9d16-524a104476ea.png">

Is there anything I can do to try and resolve this? The issue seems to persist all the time.

**Comment by logankilpatrick** [comment]
@AndreaGriffiths11 just trying to get more eyes on this

**Comment by AndreaGriffiths11** [comment]
Hey @logankilpatrick, thanks for the ping! We had a few issue a while back, but the site should be online and submissions possible. I've tested in chrome and edge. Please give it another go and let us know 😄

**Comment by logankilpatrick** [comment]
https://user-images.githubusercontent.com/35577566/154819763-46bc17cb-1930-4687-b857-76081f192cd6.mov

This still does not work for me on macOS with Safari!

**Reply by logankilpatrick** [reply]
@AndreaGriffiths11 ^

**Reply by logankilpatrick** [reply]
Bumping this @AndreaGriffiths11

**Comment by logankilpatrick** [comment]
This is still erroring : )

**Reply by shivaylamba** [reply]
<img width="1081" alt="image" src="https://user-images.githubusercontent.com/19529592/162019567-6a6840f6-b726-4a03-b8b8-42e05cdba831.png">
@logankilpatrick I am able to submit

**Comment by PaulAnsh** [comment]
Having the same issue:
![error pi ](https://user-images.githubusercontent.com/74669526/216492929-62490360-111a-4781-b9f1-5a369a303da8.png)

@shivaylamba what browser are you using in the example above?
